docs/reference/airnode/next/deployment-files/config-json.md

```diff
@@ -224,9 +224,11 @@ for some considerations.
 
 #### `options.fulfillmentGasLimit`
 
-(required) - The maximum gas limit allowed when Airnode responds to a request,
-paid by the requester. If exceeded, the request is marked as failed and will not
-be repeated during Airnode's next run cycle.
+(optional) - The maximum gas limit allowed when Airnode responds to a request,
+paid by the requester. If specified, this value will be used as the gas limit
+for the fulfillment. Note that if the gas cost exceeds the limit given, the
+request will be marked as failed and will not be retried during the next cycle.
+If not specified, Airnode will attempt to estimate the gas limit automatically.
 
 <!-- CHANGE FOR v0.12 -->
 <!--
```

Comment:
Same for the text here, which is also present for the `latest/` docs set- can you delete it there too?